Package.swift

```diff
@@ -1,18 +1,32 @@
-// swift-tools-version: 6.0
```

Comment:
Why not? 😺

Reply:
I've added a CI workflow to build the package librairies and it's not supported yet 🥲

Let's keep an eye on [this open PR adding support for Swift 6](https://github.com/swift-actions/setup-swift/pull/684).